infra/README.md

```diff
@@ -9,35 +9,57 @@ Run these steps the first time.
    see [AWS Configuration basics](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config)
    and [Where are the configuration settings stored](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html)
    for more info)
-3. To manage existing instances, install [sops](   ) to decrypt encrypted variable files in the repository.
+3. To manage existing instances, install [sops](https://github.com/getsops/sops) to decrypt encrypted variable files in the repository.
 
 ### Multi-factor authentication (MFA)
 
-If you get a 403 error when deploying or destroying resources with terraform despite having configured a valid AWS
+If you get a 403 error when running terraform despite having configured a valid AWS
 access key, you may need to set up MFA. Install both AWS CLI and jq, and make sure you have `aws` and `jq` in path. Execute the `[get-mfa-vars.sh](https://gist.github.com/mvaaltola/0abced5790401f2454444fb2ffd4acc0)` script with your AWS arn and your MFA access code,
 and finally run `. /tmp/aws-mfa-token` to temporarily set the correct MFA environment variables in your shell.
 
-## Configuration
+## Managing existing instances
 
-1. Decrypt `hame.tfvars.enc.json` by running `sops -d hame.tfvars.enc.json > hame.tfvars.json`. Alternatively, copy [tfvars.sample.json](tfvars.sample.json) to a new file called `hame.tfvars.json`
-2. Create an IAM user for CI/CD and take down the username and credentials. This can be used to configure CD deployment from Github. If CD is already configured, use existing user:
-   2. User to update lambda functions. Fill this user in `AWS_LAMBDA_USER` part in `hame.tfvars.json`. Fill credentials in Github secrets `AWS_LAMBDA_UPLOAD_ACCESS_KEY_ID` and `AWS_LAMBDA_UPLOAD_SECRET_ACCESS_KEY`.
-3. Change the values in `hame.tfvars.json` as required
+To manage existing instances, decrypt `hame-dev.tfvars.enc.json` by running `sops -d hame-dev.tfvars.enc.json > hame-dev.tfvars.json`.
+
+To make changes to instances, first check that your variables and current infra is up to date with terraform state:
+
+```shell
+terraform init
+terraform plan --var-file hame-dev.tfvars.json
```

Comment:
`terraform plan` results in few errors:

-  "db_postgres_version" must be updated  ("13.10" -"13.13) to match aws
- Syntax `vpc = true` (in vpc.tf l. 92)  is depreceated, instead should use: [](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/eip)
- `Error: reading EventBridge Rule (hame-dev-lambda-koodistot-update): AccessDeniedException: User: arn:aws:iam::631260641272:user/jaakko is not authorized to perform: events:DescribeRule on resource: arn:aws:events:eu-central-1:631260641272:rule/hame-dev-lambda-koodistot-update because no identity-based policy allows the events:DescribeRule action
│       status code: 400, request id: 3a3fc7bc-c86c-406e-8e7a-49df6c038a85
│
│   with aws_cloudwatch_event_rule.lambda_koodistot,
│   on cloudwatch.tf line 13, in resource "aws_cloudwatch_event_rule" "lambda_koodistot":
│   13: resource "aws_cloudwatch_event_rule" "lambda_koodistot" {
`
What are the policy and credentials that should be used here?